python-mini-projects/projects/Find_imdb_rating/find_IMDb_rating.py

```diff
@@ -0,0 +1,79 @@
+from bs4 import BeautifulSoup
+import requests
+import pandas as pd
+import os
+
+# Setting up session
+s = requests.session()  
+
+# List contaiting all the films for which data has to be scraped from IMDB
+films = []
+
+# Lists contaiting web scraped data
+names = []
+ratings = []
+genres = []
+
+# Define path where your films are present 
+# For eg: "/Users/utkarsh/Desktop/films"
+path = input("Enter the path where your films are: ")
+
+# Films with extensions
+filmswe = os.listdir(path)
+
+for film in filmswe:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To create a `list`, try to use `list` comprehension instead of a loop. List comprehension is the preferred way to make a list using Python, and it&#39;s simpler and easier to understand than using a loop.

[Learn more](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions)